### PR TITLE
Release 1.4.0

### PR DESCRIPTION
**Pull request by bmjen**

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -15,6 +57,7 @@ See the [release notes](https://puppet.com/docs/pdk/latest/release_notes.html) f
 **Merged pull requests:**
 
 - \(maint\) Default PDK::TEMPLATE\_REF to PDK::VERSION [\#405](https://github.com/puppetlabs/pdk/pull/405) ([rodjek](https://github.com/rodjek))
+- 1.3.2 Release Prep [\#404](https://github.com/puppetlabs/pdk/pull/404) ([HelenCampbell](https://github.com/HelenCampbell))
```
Comment (scotje) [inline CHANGELOG.md]: This seems silly but (shrug)

File: locales/pdk.pot [hunk 2]
```diff
@@ -6,11 +6,11 @@
 #, fuzzy
 msgid ""
 msgstr ""
-"Project-Id-Version: puppet development kit v1.3.1-7-g6c96455\n"
+"Project-Id-Version: puppet development kit v1.3.2-69-g30c8e60\n"
```
Comment (scotje) [inline locales/pdk.pot]: Seems like this should be 1.4.0?

File: CHANGELOG.md (hunk 1 above)
Comment (bmjen) [reply CHANGELOG.md]: Why? It was merged for the `v1.3.2` release. 🤷‍♂️

File: locales/pdk.pot (hunk 2 above)
Comment (bmjen) [reply locales/pdk.pot]: Yeah, this is weird...

Comment (DavidS) [reply locales/pdk.pot]: this is likely based off `git describe`, not lib/pdk/version.rb, so \*shrug*

Comment (scotje) [reply locales/pdk.pot]: Yeah, I don't think it actually matters

File: CHANGELOG.md (hunk 1 above)
Comment (scotje) [reply CHANGELOG.md]: In the context of a CHANGELOG communicating to humans what they need to be aware of with a new release it seems like noise, but I understand the limitations of automated CHANGELOG generation.

**Pull request merged**